compiler/rustc_attr/src/builtin.rs

```diff
@@ -463,27 +464,40 @@ pub fn cfg_matches(cfg: &ast::MetaItem, sess: &ParseSess, features: Option<&Feat
             MetaItemKind::NameValue(..) | MetaItemKind::Word => {
                 let name = cfg.ident().expect("multi-segment cfg predicate").name;
```

Comment:
```suggestion
                let ident = cfg.ident().expect("multi-segment cfg predicate");
```
Well, the span is used now.